client/src/Visit.tsx

```diff
@@ -57,11 +57,13 @@ export const Visit = (): JSX.Element => {
     const appendMeetingLinkToUrl = (): void => {
       window.history.pushState({}, document.title, window.location.href + link);
     };
-
-    appendMeetingLinkToUrl();
-
-    const meetingLinkLocator = _getTeamsMeetingLinkLocator(link);
-    setTeamsMeetingLinkLocator(meetingLinkLocator);
+    const boolValidRoomsLink = isValidRoomsLink(link);
```

Comment:
umm, name better? or since used just once we can just move it directly to the if 
if (isValidRoomsLink(link)) {
}

Reply:
done!